client/my-sites/google-my-business/select-business-type/index.js

```diff
@@ -82,22 +94,39 @@ class GoogleMyBusinessSelectBusinessType extends Component {
 	};
 
 	renderLocalBusinessCard() {
-		const { canUserManageOptions, translate } = this.props;
+		const { canUserManageOptions, googleMyBusinessLocations, siteSlug, translate } = this.props;
+		const { locationsChanged } = this.state;
 
 		let connectButton;
 
 		if ( config.isEnabled( 'google-my-business' ) && canUserManageOptions ) {
-			connectButton = (
-				<KeyringConnectButton
-					serviceId="google_my_business"
-					onClick={ this.trackCreateYourListingClick }
-					onConnect={ this.handleConnect }
-				>
-					{ translate( 'Create Your Listing', {
-						comment: 'Call to Action to add a business listing to Google My Business',
-					} ) }
-				</KeyringConnectButton>
-			);
+			if ( googleMyBusinessLocations.length > 0 && ! locationsChanged ) {
+				connectButton = (
+					<Button
+						primary
+						href={ `/google-my-business/select-location/${ siteSlug }` }
+						onClick={ this.trackCreateYourListingClick }
+					>
+						{ translate( 'Choose Google My Business Location', {
+							comment:
+								'Call to Action to choose from exisitng connected Google My Business business listings',
+						} ) }
+					</Button>
+				);
+			} else {
+				connectButton = (
+					<KeyringConnectButton
+						serviceId="google_my_business"
+						onClick={ this.trackCreateYourListingClick }
```

Comment:
We should probably fired new Track events since these new buttons convey a different purpose.